### DISCUSSION TITLE
Add filters for the LSP symbol picker

### DISCUSSION THREAD

**Discussion by LeoniePhiline** [helix-editor/helix#5791] [Ideas]
The symbol picker is largely useless to get an overview of symbols, e.g. methods in a class, as it is overflowing with small cruft (local variables).

I'd really like to see a symbol picker with a built-in symbol type filter.

**Comment by the-mikedavis** [comment]
Also see the discussion in https://github.com/helix-editor/helix/pull/4687 which talks a little about the symbol picker
